hardware/OpenCustomizationSelector/src/com/sonymobile/customizationselector/CustomizationSelectorService.java

```diff
@@ -10,47 +10,46 @@
     }
 
-    public static void evaluateCarrierBundle(Context context) {
-        synchronized (CustomizationSelectorService.class) {
-            try {
-                CSLog.logVersion(context, TAG);
-                CSLog.logSimValues(context, TAG);
+    public static synchronized void evaluateCarrierBundle(Context context) {
+        try {
+            CSLog.logVersion(context, TAG);
+            CSLog.logSimValues(context, TAG);
 
-                if (!CommonUtil.isDirectBootEnabled()) {
-                    UserManager userManager = context.getSystemService(UserManager.class);
-                    if (userManager != null && !userManager.isUserUnlocked()) {
-                        CSLog.d(TAG, "user is locked. private app data storage is not available.");
-                        return;
-                    }
+            if (!CommonUtil.isDirectBootEnabled()) {
+                UserManager userManager = context.getSystemService(UserManager.class);
+                if (userManager != null && !userManager.isUserUnlocked()) {
+                    CSLog.d(TAG, "user is locked. private app data storage is not available.");
+                    return;
                 }
+            }
 
-                Configurator configurator = new Configurator(context, CommonUtil.getCarrierBundle(context));
-                if (configurator.isNewConfigurationNeeded()) {
-                    context.getPackageManager().setComponentEnabledSetting(new ComponentName(context, CustomizationSelectorActivity.class),
-                            PackageManager.COMPONENT_ENABLED_STATE_ENABLED, PackageManager.COMPONENT_ENABLED_STATE_ENABLED);
+            Configurator configurator = new Configurator(context, CommonUtil.getCarrierBundle(context));
+            if (configurator.isNewConfigurationNeeded()) {
+                context.getPackageManager().setComponentEnabledSetting(new ComponentName(context, CustomizationSelectorActivity.class),
+                        PackageManager.COMPONENT_ENABLED_STATE_ENABLED, PackageManager.COMPONENT_ENABLED_STATE_ENABLED);
 
-                    if (isUserSetupComplete(context)) {
-                        CSLog.d(TAG, "evaluateCarrierBundle - Need to reboot, starting dialog.");
-                        context.startActivity(new Intent(Intent.ACTION_MAIN, null).addCategory(Intent.CATEGORY_HOME).addFlags(270565376));
-                    } else {
-                        CSLog.d(TAG, "evaluateCarrierBundle - Need to reboot, user setup not complete");
-                    }
+                if (isUserSetupComplete(context)) {
+                    CSLog.d(TAG, "evaluateCarrierBundle - Need to reboot, starting dialog.");
+                    context.startActivity(new Intent(Intent.ACTION_MAIN, null).addCategory(Intent.CATEGORY_HOME)
+                        .addFlags(Intent.FLAG_ACTIVITY_RESET_TASK_IF_NEEDED | Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK));
```

Comment:
There is possibly an unneeded line break here, very similar to what I saw and fixed previously in [PR 39](https://github.com/Flamefire/android_device_sony_yoshino-common/pull/39/files#diff-179d09fcdc44db77e283449d7db690744bb735fdf6f79f6ab6f85cdb6010d128).
Does this code work?

Reply:
Searching through framework code and GitHub the this is as intented (how punny...). However here it is inconsistent: `addCategory` should have been on a new line already.    
Also the preferred style seems to be to create an `Intent` variable first, then pass that to `startActivity`. Done in ca8df4d and related 13cb0e6